lib/framework/action.dart

```diff
@@ -741,7 +741,9 @@ enum ActionType {
   getPhoneContacts,
   checkPermission,
   saveToKeychain,
+  saveToKeychainWithGroupId,
```

Comment:
why? our Actions will balloon to an unreasonable number. GroupId is simply another attribute